src/index.html

```diff
@@ -36,10 +38,10 @@ <h1>What's Cookin'</h1>
         <button type="button" name="back-to-main" class="back-to-main-btn" id="backToMainBtn">Back to Main</button>
         <p id="recipeTitle">Elvis Pancakes</p>
         <img id="indRecipeImage" src="https://spoonacular.com/recipeImages/741603-556x370.jpeg" alt="Elvis Pancakes">
-        <h3 id="addToFavoriteList">☆ Add to Favorite List</h3>
-        <h3 id="onFavoriteList">★ On Favorite List</h3>
-        <h3 id="addToCookingList">☐ Add to Cooking List</h3>
-        <h3 id="onCookingList">☑︎ On Cooking List</h3>
+        <h3 class="list-btns" id="addToFavoriteList">☆ Add to Favorite List</h3>
```

Comment:
Hiding and showing buttons based on whether or not the recipe is contained in the array.